src/python/gem5/prebuilt/demo/x86_demo_board.py

```diff
@@ -77,23 +83,68 @@ def __init__(self):
             "real-world system. Use with caution."
         )
 
-        memory = SingleChannelDDR3_1600(size="2GiB")
+        # The other demo boards have 4 GiB of memory, but X86Board can only
+        # support up to 3 GiB.
+        memory = DualChannelDDR4_2400(size="3GiB")
         processor = SimpleProcessor(
-            cpu_type=CPUTypes.TIMING, isa=ISA.X86, num_cores=4
+            cpu_type=CPUTypes.TIMING, isa=ISA.X86, num_cores=2
         )
-        cache_hierarchy = MESITwoLevelCacheHierarchy(
-            l1d_size="32KiB",
-            l1d_assoc=8,
-            l1i_size="32KiB",
-            l1i_assoc=8,
-            l2_size="1MiB",
-            l2_assoc=16,
-            num_l2_banks=1,
+
+        cache_hierarchy = PrivateL1SharedL2CacheHierarchy(
+            l1d_size="64KiB", l1i_size="64KiB", l2_size="1MiB"
         )
 
         super().__init__(
-            clk_freq="3GHz",
+            clk_freq="1.4GHz",
```

Comment:
This frequency seems low and it's going to make simulations take longer. 3GHz is pretty reasonable.

Reply:
This is mostly to keep it similar the riscv and arm demo boards so they are somewhat comparable. a 3GHz RISC-V, for example, seems unrealistic

Reply:
The frequency depends on the microarchitecture, not the architecture (ISA). 1.4 GHz is too slow for modern devices.

Reply:
I meant more that i felt the boards demo boards should be similar rather than individually true-ish representations of what an X86/RISCV/ARM system is. I thought this would make it easier so we can compare their performance fairly.

I'm not against making this more realistic I just wanted you to know this makes the demoboards less alike. I'll make a patch putting this at 3GHz